Gem/Code/Source/Components/NetworkWeaponsComponent.cpp

```diff
@@ -375,7 +375,7 @@ namespace MultiplayerSample
                 }
 
                 // Validate the proposed start position is reasonably close to the related bone
-                float startPositionClampRange = .5f;
+                float startPositionClampRange = 1.f;
```

Comment:
This really is starting to look like this should be a cvar.

Reply:
I suppose it might as well. This change was because I did not account for the range change while jumping but if the model changes it'll be worth adjusting.

Reply:
+1 on the cvar.. any time you have a static magic number like this, may as well make it tunable.